mlibrary/search
# Updating `history` and `react-router-dom`.

# Overview
`react-router-dom` has not been updated because it introduces breaking changes. It is also only compatible with `history` versions `5` and up, so that dependency has also not been updated.

This pull request finally updates the two dependencies to make this repository fully up to date.

## Testing
List instructions on how to test the pull request. Some examples:

- Run the tests to make sure they pass (`npm run test`).
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- _How to use the feature_.
